tests/frameworks/OSXShims/src/LoggingNative.mm

```diff
@@ -15,44 +15,92 @@
 //******************************************************************************
 
 #import <LoggingNative.h>
+#import <Starboard/SmartTypes.h>
 #import <Foundation/NSString.h>
 
 #import <cstdarg>
 
+// NOTE: this function is assumed to only run in Little Endian environments like OSX
+static NSStringEncoding _getWcharEncoding() {
+    switch (sizeof(wchar_t)) {
+        case 2:
+            return NSUTF16LittleEndianStringEncoding;
+        case 4:
+            return NSUTF32LittleEndianStringEncoding;
+        default:
+            assert(false);
+    }
+}
+
+static NSStringEncoding getWcharEncoding() {
```

Comment:
maybe? I don't really care one way or another but I don't think many other devs are going to stumble on WCHAR_ENCODING. I also don't think there are big endian OSX environments? We can chat in person about what we want.

---
In reply to: [74168895](https://github.com/Microsoft/WinObjC/pull/740#discussion_r74168895) [](ancestors = 74168895)